packages/mdc-ripple/README.md

```diff
@@ -182,6 +182,23 @@ ripple.
 MDCRipple.attachTo(document.querySelector('.surface'));
 ```
 
+### Ripple API
+
+The component allows for programmatic activation / deactivation of the ripple, for complex interaction between deeply
+tied components. This is used for making form field labels trigger the ripples in their corresponding input elements,
+for example.
+
+#### MDCRipple.activate()
+
+Triggers an activation of the ripple (the first stage, which happens on a down event). It is represented visually as if
+coming from a keyboard event.
+
+#### MDCRipple.deactivate()
+
+Triggers a deactivation of the ripple (the second stage, which happens on an up event). It is represented visually as if
```

Comment:
could we say "which happens when the ripple surface is engaged via interaction, such as a `mouseup` or a `pointerup` event instead of "which happens on an up event"?